services/placement/options.go

```diff
@@ -50,6 +50,8 @@ func (o deploymentOptions) SetMaxStepSize(stepSize int) DeploymentOptions {
 	return o
 }
 
+func defaultTimeNanosFn() int64 { return 0 }
```

Comment:
I was gonna handle the case in shard.SetCutoffNanos() because when you get a 0 cutoffNanos from proto, and set that value on a shard, the shard needs to know it means maxInt64 as well, so I'm just going to use this same defaultTimeNanosFn.

This means you can never set a CutoffNanos with 0, but I think that is ok?